ractor_actors/src/net/tcp/listener.rs

```diff
@@ -3,98 +3,88 @@
 // This source code is licensed under both the MIT license found in the
 // LICENSE-MIT file in the root directory of this source tree.
 
-//! Tcp listener actor
+//! TCP Server to accept incoming sessions
```

Comment:
in general most of the edits here seem to be a loss of flexibility and really mostly a no-op change? I'm not sure if this is a consequence of the clone of logic from ractor_cluster, but this version was written after and is more generic and flexible by design.